### PR TITLE
refactor: set admin as default role for script and update documentation

### PR DESCRIPTION
**Pull request by anversa-pro**
## Issue ticket number and link

https://trello.com/c/3Rdto66a

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics?
- [ ] Will this be part of a product update? If yes, please write one phrase about this update.

## Pull request type

Please check the type of change your PR introduces:

- [ ] :sparkles: Feature
- [x] :bug: Bugfix
- [ ] :pencil2: Fixing typos
- [ ] :lipstick: :art: Code style update (formatting, renaming)
- [ ] :recycle: Refactoring (no functional changes, no api changes)
- [ ] :zap: Performance
- [x] :pencil: Documentation content changes
- [ ] :white_check_mark: Unit testing
- [ ] :ok_hand: Updating due to code review suggest
- [ ] :rocket: Release: v
- [ ] :wrench: Add or update configuration files
- [x] :hammer: Add or update development scripts
- [ ] :bricks: Infrastructure related changes

## What is the current module?

- The script for retrieving authorization tokens is currently configured for a test user.

## What is the new behavior?

- New features have been introduced, granting authorization for specific app operations exclusively to admin users. Consequently, the script's default role has been updated to 'admin.' Nonetheless, the option to retrieve a user token remains accessible.

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: scripts/firebase-auth-test.js [hunk 1]
```diff
@@ -27,12 +25,23 @@ const getFirebaseApp = () => {
 
 const runAuthentication = async () => {
   const auth = getAuth(getFirebaseApp())
+  const role = process.argv[2] || 'admin'
+  let email = process.env.ADMIN_USER_EMAIL
```
Comment (jose-sal-y-rosas) [inline scripts/firebase-auth-test.js:29]: Please, remove the use of these from env vars. Suggestion here: In this PR o create a new task for removing it

File: scripts/firebase-auth-test.js [hunk 2]
```diff
@@ -27,12 +25,23 @@ const getFirebaseApp = () => {
...
+  let password = process.env.ADMIN_USER_PWD
```
Comment (jose-sal-y-rosas) [inline scripts/firebase-auth-test.js:30]: Please, remove the use of these from env vars. Suggestion here: In this PR o create a new task for removing it

File: scripts/firebase-auth-test.js [hunk 3]
```diff
@@ -27,12 +25,23 @@ const getFirebaseApp = () => {
...
+
+  if (role === 'user') {
+    email = process.env.TEST_USER_EMAIL
```
Comment (jose-sal-y-rosas) [inline scripts/firebase-auth-test.js:33]: Please, remove the use of these from env vars. Suggestion here: In this PR o create a new task for removing it

File: scripts/firebase-auth-test.js [hunk 4]
```diff
@@ -27,12 +25,23 @@ const getFirebaseApp = () => {
...
+    password = process.env.TEST_USER_PWD
```
Comment (jose-sal-y-rosas) [inline scripts/firebase-auth-test.js:34]: Please, remove the use of these from env vars. Suggestion here: In this PR o create a new task for removing it

File: README.md [hunk 5]
```diff
@@ -221,3 +229,67 @@ TBD
...
+- An `.env` file containing your Firebase configuration and user credentials
+  - **Configuration:**
+    - `AUTH_API_KEY`
+    - `AUTH_DOMAIN`
+    - `AUTH_PROJECT_ID`
+    - `AUTH_STORAGE_BUCKET`
+    - `AUTH_MESSAGING_SENDER_ID`
+    - `AUTH_APP_ID`
+    - `AUTH_MEASUREMENT_ID`
+  - **Credentials:**
```
Comment (jose-sal-y-rosas) [inline README.md:254]: This section should be removed according to the comments below

**Review by jose-sal-y-rosas — CHANGES_REQUESTED** [review]
Please review feedback

File: scripts/firebase-auth-test.js (hunk 1 above)
Comment (anversa-pro) [reply scripts/firebase-auth-test.js]: The environment variables for roles have been removed from the current PR. A new task will be created to handle the removal of these variables from the entire project.

File: scripts/firebase-auth-test.js (hunk 3 above)
Comment (anversa-pro) [reply scripts/firebase-auth-test.js]: The environment variables for roles have been removed from the current PR. A new task will be created to handle the removal of these variables from the entire project.

File: README.md (hunk 5 above)
Comment (anversa-pro) [reply README.md]: The section has been adjusted in accordance with the feedback provided, now accurately reflects the current usage.

File: scripts/firebase-auth-test.js (hunk 2 above)
Comment (anversa-pro) [reply scripts/firebase-auth-test.js]: The environment variables for roles have been removed from the current PR. A new task will be created to handle the removal of these variables from the entire project.

File: scripts/firebase-auth-test.js (hunk 4 above)
Comment (anversa-pro) [reply scripts/firebase-auth-test.js]: The environment variables for roles have been removed from the current PR. A new task will be created to handle the removal of these variables from the entire project.

**Review by jose-sal-y-rosas — APPROVED** [review]
👍🏼

File: scripts/firebase-auth-test.js [hunk 6]
```diff
@@ -4,6 +4,17 @@ const { initializeApp } = require('firebase/app')
 
 dotenv.config()
 
+const roleSettings = {
+  admin: {
+    email: 'adminuser@chupito.com',
```
Comment (FelipeLosada1996) [inline scripts/firebase-auth-test.js:9]: Si estos serán los usuarios admin, dejémoslos como una variable que las personas puedan cambiar sin necesidad de tocar el código

**Review by FelipeLosada1996 — APPROVED** [review]

File: scripts/firebase-auth-test.js (hunk 6 above)
Comment (anversa-pro) [reply scripts/firebase-auth-test.js]: Se configuró un JSON para almacenar las credenciales e importarlas al script.

**Review by jose-sal-y-rosas — APPROVED** [review]
👍🏼

**Pull request merged**